src/2d/shallow/topo_module.f90

```diff
@@ -401,8 +404,12 @@ end subroutine set_topo_for_dtopo
 
     subroutine read_topo_file(mx,my,topo_type,fname,topo)
 
+#ifdef NETCDF
+        use netcdf
```

Comment:
What is this idef checking?  I get an error when I try to compile in `examples/tsunami/chile2010` without setting any preprocessing flag.

```

        use netcdf
                  1
Fatal Error: Can't open module file 'netcdf.mod' for reading at (1): No such file or directory
```